package/base-files/files/lib/upgrade/common.sh

```diff
@@ -91,7 +91,7 @@ get_image() { # <source> [ <command> ]
 		esac
 	fi
 
-	cat "$from" 2>/dev/null | $cmd
+	$cmd "$from" 2>/dev/null
```

Comment:
This will suppress errors for $cmd, while the first one only suppressed errors from cat.

Reply:
I dropped the redirection.

Reply:
Since this is upgrade code, and this is still a behavior change, this should be reviewed and tested carefully.